access/types/forms.py

```diff
@@ -292,6 +292,14 @@ def add_field(self, i, config, field_class, widget_class,
                 field.correct = config['model']
             elif config.get('correct', False):
                 field.correct = config['correct']
+            if 'options' in config:
+                neutral = []
+                a = 0
+                for opt in config['options']:
+                    if opt.get('correct') == 'neutral':
+                        neutral.append(self.option_name(i, opt))
```

Comment:
Either the variable `a` is defined for no reason or `self.option_name(i, opt)` should use `a` instead of `i`.

Reply:
I changed it to use `a`.